qa-pipelines/set-pipeline

```diff
@@ -17,31 +17,35 @@ Available options:
     -t, --target=       Set fly target concourse
     -b, --branch=       Use a specific branch or tag. Defaults to the current branch
     --nightly           Deploy pipeline that triggers when new nightly builds are released
+    --pr=               Deploy pipeline that tests a build from a specific PR
```

Comment:
why there is a `=` ?

Reply:
ok, because it takes the PR value `s3_config_prefix=prs/PR-${2}-`
Please add usage in Readme along with an example.